src/MemoryTable.js

```diff
@@ -16,8 +16,23 @@ export let memoryService = (records, { schema, debug } = {}) =>
     providers: { memory: { ...memory, types: types() } },
   })
 
+export let useMemory = ({ schema, debug } = {}) => {
+  let [storage] = React.useState({})
+  let [service] = React.useState(
+    Contexture({
```

Comment:
`useState` can take a fn to only compute on first render, which we should do here. Also this looks exactly like the `memoryService` function in this file.

Also further: it looks like `memoryService` is used only in stories and the stories themselves could use `useMemory` instead so I don't think there's a reason for `memoryService` to exist anymore.

Reply:
Refactored into useMemoryTree